addons/mail/static/src/core/common/chat_hub.js

```diff
@@ -33,10 +33,7 @@ export class ChatHub extends Component {
         this.onResize();
         useExternalListener(browser, "resize", this.onResize);
         useEffect(() => {
-            if (
-                this.chatHub.actuallyFolded.length &&
-                this.store.channels?.status === "not_fetched"
-            ) {
+            if (this.chatHub.folded.length && this.store.channels?.status === "not_fetched") {
```

Comment:
I thought we wanted to enforce length check against 0

Reply:
It behaves the same, honestly this is more subjective than anything else.
I just renamed `actuallyFolded` to `folded` in this diff